src/software/world/robot.h

```diff
@@ -48,18 +50,20 @@ class Robot
      * @param timestamp The timestamp at which the robot was observed to be in the given
      * state. The timestamp must be >= the robot's latest update timestamp
      */
-    void updateState(const Point &new_position, const Vector &new_velocity,
-                     const Angle &new_orientation,
-                     const AngularVelocity &new_angular_velocity,
-                     const Timestamp &timestamp);
+    void updateCurrentState(const Point &new_position, const Vector &new_velocity,
```

Comment:
semi-nit: What are everyone's thoughts on `updateState` vs `updateCurrentState`? Seeing it changed to `updateCurrentState` threw me off for a second because I thought there might also be a `updateState` function and wasn't sure what the difference would be. IMO `updateCurrentState` almost implied the current state is mutated directly? This might all be due to my previous expectations and just being used to the old names, so would really like some input here.

Reply:
@garethellis0 @jonathanlew @scveloso thoughts on this?

Reply:
I like `updateState`, not a fan of `updateCurrentState`.

Reply:
Yeah I feel like ```updateState``` kind of already implies update the current state

Reply:
Agreed

Reply:
changed updateCurrentState to updateState